README.md

```diff
@@ -28,31 +28,134 @@ flag is highly recommended for those new to [pip](https://pip.pypa.io).
 A PEX executable (Windows not supported) and source releases are
 [here](https://github.com/planetlabs/planet-client-python/releases/latest).
 
-## Documentation
+## API Key
 
-Online documentation:
-https://planetlabs.github.io/planet-client-python/index.html
+The planet API requires an account for use. [Signup here](https://www.planet.com/explorer/?signup).
 
-Documentation is also provided for download
-[here](https://github.com/planetlabs/planet-client-python/releases/latest).
+This can be provided via the environment variable `PL_API_KEY` or the flag `-k` or `--api-key`.
 
+Using `planet init` your account credentials (login/password) can be used to obtain the api key.
 
-## Development
+## Quick Start
 
-To contribute or develop with this library, see
-[CONTRIBUTING](https://github.com/planetlabs/planet-client-python/CONTRIBUTING.md)
+### Python Library
 
+The python library is asynchronous, which greatly speeds up many interactions
+with the planet API. 
 
-## API Key
+```python
+import asyncio
+import os
+from planet.api import APlanetSession, AOrdersClient
```

Comment:
Does something exist at the level of `import planet` for a user to use instead of nested modules?

Reply:
In v1, there was nothing at the level of `import planet`. I haven't changed that yet in v2, but I'd like to. My only blocker is that I haven't tackled the CLI yet, so I don't know if that would cause any conflicts with the CLI. But honestly, it is unlikely to cause issues and the usability advantage would make any extra work with the CLI, if it does come up, worth it.

Reply:
+1 for enabling top level imports

Reply:
Awesome! We are all on the same page! I'm on it.

Reply:
https://github.com/planetlabs/planet-client-python/pull/240/commits/a33ac22592eb00829cf188379e1f88c434a17928